source/transfer_list.rst

```diff
@@ -902,5 +902,49 @@ This manifest contains the SPMC attribute node consumed by the SPMD
      - hdr_size
      - Holds a SPMC manifest image in DT format.
 
+**Read-Write Memory Layout Entry Layout (XFERLIST_RW_MEM_LAYOUT64)**
+
+This entry type holds a structure that describes the layout of a read-write
+memory region. TF-A defines this structure and uses it to convey to BL2 the
+extent of memory it has available to perform read-write operations on. BL2 maps
+the memory described by the layout into its memory map during platform setup. If
+other memory types are required (i.e.  read-only memory) separate TE's should be
+defined.
```

Comment:
Would it be useful to generalise this to include any 'next phase'??

Reply:
What do you mean? I refer to its usage in BL2 here only as a concrete example, but it very well could be used in other places.

Reply:
OK, then please put 'For example, ...' in a separate paragraph so it is clear